locations/kotti.yml

```diff
@@ -0,0 +1,69 @@
+---
+
+location: kotti
+location_nice: Skalitzer Straße 134, 10999 Berlin
+latitude: 52.4992763
+longitude: 13.4195371
```

Comment:
The location used to be differen. It this really mounted in a different spot now? If not, please correct:

![image](https://github.com/user-attachments/assets/24152f9e-1635-422a-91d3-f2b268033efe)